tests/base/quadrature_push_back.output

```diff
@@ -0,0 +1 @@
+
```

Comment:
In cases where the correctness of the test is determined by whether or not an exception is thrown, we typically add
```
  deallog << "OK" << std::endl;
```
as the last line of the `test_...()` function. This then shows up in the output file if the test succeeds, and produces a visual record when a test fails.

Reply:
Added